pkgs/by-name/ba/bartender/package.nix

```
@@ -32,17 +39,37 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru.updateScript = lib.getExe (writeShellApplication {
+    name = "bartender-update-script";
+    runtimeInputs = [
+      curl
```

```suggestion
      cacert
      curl
```
*Pretty sure* we also need this if running the script purely?

---

From my understanding update scripts are suppose to be ran with https://github.com/NixOS/nixpkgs/blob/master/maintainers/scripts/update.nix *(e.g `APP=bartender; nix-shell --pure ./maintainers/scripts/update.nix --argstr package "$APP" --argstr commit true --argstr skip-prompt true`)*

And in `update.nix` we have this line: https://github.com/NixOS/nixpkgs/blob/0311f6c40e0a7ea6bb1e967b5a607871c654d5f6/maintainers/scripts/update.nix#L234

Which gives us the cacert already

---

> Correct me if I'm wrong, but from my understanding, update scripts are supposed to be ran with https://github.com/NixOS/nixpkgs/blob/master/maintainers/scripts/update.nix

They can be, but they can (or well, should) also be run on their own. Sorta a nitpick, but I think it's a nice thing to account for